spring-cloud-dataflow-server-local/src/main/resources/META-INF/kafka-apps.properties

```diff
@@ -37,3 +37,8 @@ sink.tcp=maven://org.springframework.cloud.stream.app:tcp-sink-kafka:@registry.a
 sink.throughput=maven://org.springframework.cloud.stream.app:throughput-sink-kafka:@registry.app.defaultVersion@
 sink.websocket=maven://org.springframework.cloud.stream.app:websocket-sink-kafka:@registry.app.defaultVersion@
 task.timestamp=maven://org.springframework.cloud.task.app:timestamp-task:@registry.app.defaultVersion@
```

Comment:
Given these task app starters don't have any `binder` dependency, can we move them to a common `properties` file named `task-apps.properties` or something?